pkg/descheduler/evictions/evictions.go

```diff
@@ -184,6 +184,7 @@ func evictPod(ctx context.Context, client clientset.Interface, pod *v1.Pod, poli
 	err := client.PolicyV1beta1().Evictions(eviction.Namespace).Evict(ctx, eviction)
 
 	if apierrors.IsTooManyRequests(err) {
+		metrics.TotalPodsSkipped.With(map[string]string{"result": "total pods skipped so far", "namespace": pod.Namespace}).Inc()
```

Comment:
Setting the `result` to `total pods skipped so far` is redundant here given the metric's name is `TotalPodsSkipped`. I wonder if it would make more sense to set the `result` to `pod skipped due to TooManyRequests error`?